bin/oneoff/accessibility/accessibility_checker.rb

```diff
@@ -39,10 +39,9 @@ def main
     # Sort from most to least frequent errors
     error_results = error_results.sort_by {|_, count| -count}
 
-    puts ""
-    puts "Most Common Errors for #{url}:"
+    ChatClient.message 'wcag-checker', "Most Common Errors for #{url}:"
     error_results.each do |error, count|
-      puts "  #{error} - #{count}"
+      ChatClient.message 'wcag-checker', "  #{error} - #{count}"
```

Comment:
[Not blocking] Is it possible to make the error messages more human-readable? i.e. "Clickable Div". Or perhaps just add a link to a doc that makes it easy to find the error?

Reply:
This is a good call! There's a lot of rules possible so I'm not sure how to make more human-readable messages, but I think I should be able to programmatically link to WCAG rules. I'm going to attempt in a follow-up once I see if there are other changes I need to make for this to show up nicely in Slack.